package.json

```diff
@@ -157,6 +159,10 @@
     "web-ext-types": "^3.2.1",
     "xml-js": "^1.6.11"
   },
+  "dependenciesComments": {
```

Comment:
@CedrikNikita @martinkaintas what do you think about this way of adding our custom comments to installed packages? I feel that it might be helpful as we sometimes add packages that we need to update or change in some conditions.

Reply:
I think we would only benefit from the approach, good suggestion.